features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -177,9 +177,10 @@ nsapi_error_t UBLOX_AT_CellularStack::socket_connect(nsapi_socket_t handle, cons
     _at.write_string(addr.get_ip_address());
     _at.write_int(addr.get_port());
     _at.cmd_stop_read_resp();
+    nsapi_error_t err = _at.get_last_error();
```

Comment:
could do in one line
nsapi_error_t err = _at.unlock_return_error();
insted of 
nsapi_error_t err = _at.get_last_error();
_at.unlock();